src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.html

```diff
@@ -0,0 +1,343 @@
+<div cdsCol
+     [columnNumbers]="{ md: 4 }"
+     *ngIf="orchStatus$ | async as orchStatus">
+  <form name="smbForm"
+        #formDir="ngForm"
+        [formGroup]="smbForm"
+        novalidate>
+    <div i18n="form title"
+         class="form-header">
+      {{ action | titlecase }} {{ resource | upperFirst }}
+    </div>
+
+    <!-- Cluster Id -->
+    <div class="form-item">
+      <cds-text-label
+        for="cluster_id"
```

Comment:
```suggestion
        labelInputID="cluster_id"
```
See this: https://github.com/carbon-design-system/carbon-components-angular/blob/v5.56.2/src/input/text-input-label.component.ts#L38